packages/devtools_app/lib/src/shared/status_line.dart

```diff
@@ -242,12 +238,12 @@ class IsolateSelector extends StatelessWidget {
               onChanged: isolateManager.selectIsolate,
               isDense: true,
               items: isolates.map(
-                (IsolateRef ref) {
+                (IsolateRef? ref) {
                   return DropdownMenuItem<IsolateRef>(
                     value: ref,
                     child: Row(
                       children: [
-                        ref.isSystemIsolate
+                        ref!.isSystemIsolate!
```

Comment:
what if `ref` is null? we should either handle that case or make isolateManager.isolates.value have type `<IsolateRef>[]` instead of `<IsolateRef?>[]`. Prefer the latter unless we are explicitly adding a null value to this list somewhere.

Reply:
done